AppBuilder/platform/process/tasks/ABProcessTaskUserExternal.js

```diff
@@ -4,4 +4,21 @@ const ABProcessTaskUserExternalCore = require("../../../core/process/tasks/ABPro
 
 module.exports = class ABProcessTaskUserExternal extends (
    ABProcessTaskUserExternalCore
-) {};
+) {
+   constructor(attributes, process, AB) {
+      super(attributes, process, AB);
+
+      this.AB.on("ab.inbox.update", (item) => {
```

Comment:
instead of having this listen for `ab.inbox.update`, I think you should have the `ui/portal_work_inbox.js`.`init()` function listen for the `ab.inbox.update` and then remove the inbox item from it's current list of items.